src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_RADIUS = 10;
+
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private Random index = new Random();
+
+    private ColorSupplier color = new ColorSupplier();
```

Comment:
1. Please remove excessive empty lines
2. Name index is confusing as for me, as used to generate sides, not indexes
```suggestion
    private static final double DEFAULT_RADIUS = 10;
    private static final Color DEFAULT_COLOR = Color.WHITE;

    private Random random = new Random();
    private ColorSupplier color = new ColorSupplier();
```